ipapython/ipautil.py

```diff
@@ -127,6 +127,17 @@ def __init__(self, addr):
         super(UnsafeIPAddress, self).__init__(addr,
                                               flags=self.netaddr_ip_flags)
 
+    def __getstate__(self):
+        state = {
+            '_net': self._net,
+            'super_state': super(UnsafeIPAddress, self).__getstate__(),
+        }
+        return state
+
+    def __setstate__(self, state):
+        super(UnsafeIPAddress, self).__setstate__(state.pop('super_state'))
```

Comment:
You don't really have to `.pop()` anymore, as `state` is no longer passed further.